specs/spec_v1.0.md

```diff
@@ -1,322 +0,0 @@
-# **Summary**
```

Comment:
I don't mind to remove this file but as I recall @cmihai used it for the reference. Would be good to ask his opinion.

Reply:
That was just to understand the commit messages. I don't need the spec right now.